backend/index.ts

```diff
@@ -26,6 +26,7 @@ import {
   setApplicationInfo,
   submitApplicationInfo,
 } from "./routes/application_info";
+import {createEventPushSubscription, deleteEventPushSubscription, getEventSubscriptions} from "./routes/event_subscriptions"
```

Comment:
Super nit: add space between open/close brackets
```suggestion
import { createEventPushSubscription, deleteEventPushSubscription, getEventSubscriptions } from "./routes/event_subscriptions"
```